code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -403,7 +407,25 @@
 //Applies brute and burn damage to the organ. Returns 1 if the damage-icon states changed at all.
 //Damage will not exceed max_damage using this proc
 //Cannot apply negative damage
-/obj/item/bodypart/proc/receive_damage(brute = 0, burn = 0, blocked = 0, updating_health = TRUE, required_bodytype = null, wound_bonus = 0, bare_wound_bonus = 0, sharpness = NONE, attack_direction = null)
+/**
```

Comment:
you should really include these non-DMdoc comments into your new documentation or just x-them out entirely

Reply:
I actually did include them, I just reworded them and forgot to delete it.